samples/charts/financial-chart/axis-types/Services/StocksUtility.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Samples.Services
+{
+    public class StocksUtility
+    {
+        public static int intervalDays = 1;
+        public static int intervalHours = 0;
+        public static int intervalMinutes = 0;
+
+        public static double priceStart = 200;
+        public static double priceRange = 1;
+        public static double volumeRange = 1000;
+        public static double volumeStart = 20000000;
+
+        public static List<StockItem> GetStocksFrom(DateTime dateEnd, int years)
+        {
+            var dateStart = dateEnd.AddYears(-years);
+            return GetStocksBetween(dateStart, dateEnd);
+        }
+
+        public static List<StockItem> GetStocksItems(int points)
+        {
+            intervalDays = 0;
+            intervalHours = 1;
+            intervalMinutes = 0;
+
+            var today = DateTime.Today;
+            var year = today.Year;
+            var dateEnd = new DateTime(year, 11, 1);
+            var dateStart = dateEnd.AddHours(-points);
+            return GetStocksBetween(dateStart, dateEnd);
+        }
+
+        public static List<StockItem> GetStocksBetween(DateTime dateStart, DateTime dateEnd)
+        {
+
+            var interval = intervalDays * 24 * 60;
+            interval += intervalHours * 60;
+            interval += intervalMinutes;
+
+            var time = dateStart.AddDays(0);
+            var v = volumeStart;
+            var o = priceStart;
+            Random random = new Random();
```

Comment:
the random variable should be a static field in this class instead of local variable of this function

Reply:
make sure you make the same corrections in StocksUtility.cs in other samples

Reply:
pending